mosfit/modules/observables/photometry.py

```diff
@@ -70,6 +72,8 @@ def __init__(self, **kwargs):
         self._filter_integrals = [0.0] * self._n_bands
         self._average_wavelengths = [0.0] * self._n_bands
         self._band_offsets = [0.0] * self._n_bands
+        FLUX_STD = 3631 * u.Jy.cgs.scale / u.Angstrom.cgs.scale * C_CGS
+        ANG_CGS = u.Angstrom.cgs.scale
```

Comment:
Move these defs into the class itself (like in `Blackbody`) and refer to them as `self.FLUX_STD` e.g.